<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Connect</class>
 <widget class="QWidget" name="Connect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connecting menu</string>
  </property>
  <property name="inputMethodHints">
   <set>Qt::ImhDigitsOnly</set>
  </property>
  <widget class="QLineEdit" name="addressEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="text">
    <string>127.0.0.1</string>
   </property>
   <property name="placeholderText">
    <string>127.0.0.1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="portSpinBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>8080</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>adav</string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="placeholderText">
    <string>Gordon</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
